hivemq-edge/src/frontend/src/modules/ProtocolAdapters/components/drawers/AdapterInstanceDrawer.spec.cy.tsx

```diff
@@ -68,11 +68,38 @@ describe('AdapterInstanceDrawer', () => {
         isNewAdapter={true}
         isOpen={true}
         isSubmitting={false}
-        onSubmit={cy.stub().as('onSubmit')}
-        onClose={cy.stub().as('onClose')}
+        onSubmit={cy.stub()}
+        onClose={cy.stub()}
       />
     )
     cy.checkAccessibility()
     cy.percySnapshot('Component: AdapterInstanceDrawer')
   })
+
+  describe.only('Custom Templates', () => {
```

Comment:
issue(non-blocking): watch out only this block will be executed

```suggestion
  describe('Custom Templates', () => {
```